prometheus/client.go

```diff
@@ -39,6 +40,7 @@ type Client struct {
 	ClientInterface
 	p8s api.Client
 	api prom_v1.API
+	ctx context.Context
```

Comment:
Context is mandatory but we are not using, instead to create a new empty one with Background() call, we can store it at client level for future uses.